README.org

```diff
@@ -990,6 +993,14 @@ The =embark-consult= package provides the following exporters:
   to navigate among matches, and, if you install the [[http://github.com/mhayashi1120/Emacs-wgrep/raw/master/wgrep.el ][wgrep]] package,
   you can use it to edit the matches in place.
 
+- An alternate exporter for =consult-line= allows to use it with grep buffer
+  and benefit from wgrep and its multi-line results editing capabilities.
```

Comment:
@artemkovalyov Can you please clarify what you meant by multi-line here. I suspect @oantolin and I are not the only ones who are not aware of this feature of wgrep.

Reply:
@minad, I'll update the docs and add some wiki after we agree on the implementation. It's a pre-requisite.